qark/qarkMain.py

```diff
@@ -746,8 +754,35 @@ def main():
     #find all R.java files
     common.xml_files=common.find_xml(common.sourceDirectory)
 
-    if common.interactive_mode:
-        stop_point = raw_input("Press ENTER key to begin Static Code Analysis")
+    '''
+        Decompile switch to take user input to stop after decompilation or continue with static code analysis
+    '''
+
+    if common.source_or_apk == 1:
+        try:
+            if common.interactive_mode:
+                print common.term.cyan + common.term.bold + str(common.config.get('qarkhelper','DECOMPILE_CHOICE')).decode('string-escape').format(t=common.term)
+                decompile_choice=int(raw_input(common.config.get('qarkhelper','ENTER_YOUR_CHOICE')))
+                if decompile_choice == 1:
+                    common.exitClean()
+                    exit()
+                if decompile_choice not in (1,2):
```

Comment:
Not sure what `2` is for, but if it is a true/false flag we won't need to have the user specify their value on the command line because it is `False` by default if not supplied